pkg/network/openshift_sdn.go

```diff
@@ -239,7 +242,7 @@ func clusterNetwork(conf *operv1.NetworkSpec) (string, error) {
 		},
 
 		PluginName:       sdnPluginName(c.Mode),
-		Network:          networks[0].CIDR,
+		Network:          strings.Join(cidrs[:], ", "),
 		HostSubnetLength: networks[0].HostSubnetLength,
```

Comment:
if I do change the Network field as proposed, it sort of makes the HostSubnetLength look wrong.